app/controllers/api/v2/compliance/policies_controller.rb

```diff
@@ -1,6 +1,10 @@
 module Api::V2
   module Compliance
     class PoliciesController < ::Api::V2::BaseController
+      include Foreman::Controller::SmartProxyAuth
+
+      add_smart_proxy_filters :content
```

Comment:
I don't see this feature defined in smart_proxy_openscap, isn't content feature added by pulp node? Should we  have our own feature perhaps?